script/mirror

```diff
@@ -14,22 +14,35 @@ test_mirrored() {
   curl -qsSfIL "$RUBY_BUILD_MIRROR_URL/$1" >/dev/null 2>&1
 }
 
-compute_sha2() {
-  local output="$(openssl dgst -sha256)"
-  echo "${output##* }" | tr '[A-Z]' '[a-z]'
+compute_checksum() {
+  local digest="$1"
```

Comment:
I feel like this doesn't need a local variable, but if it does, it's better that it's called something like `type` rather than ambiguous "digest".